source/getting_started/on-container.rst

```diff
@@ -0,0 +1,84 @@
+.. _caas-on-container:
+
+Run |C| in Container
+====================
+
+Introduction
+------------
+
+The goal of **Celadon in Container (CIC)** feature is to run the |C| Android image in a Docker container, so that you can run the image on Linux devices through Docker tools, and run Android applications in it.
+
+CIC should be able to run on modern PCs with Intel® processors which have Intel 6th Generation or later GPU integrated. The |NUC| model `NUC7i7BNH <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i7bnh.html>`_ and model `NUC7i5BNH <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i5bnh.html>`_ are recommended to try out the CIC features.
+
+.. note::
+   The CIC is currently of Version 0.5, whcih provides a preview of the feature for pilot and development purposes. Some features such as Trusty, Verified Boot, and OTA update are not included in this preview version. Those features is planned for the upcoming releases.
```

Comment:
- typo here: "whcih"
- "Intel® processors which have Intel 6th Generation or later GPU integrated" should be "Intel® 6th generation or later processors with integrated GPU"
- "Those features is planned" should be "Those features are planned"

Reply:
@joez, fixed and thanks.